lxd/cluster/membership.go

```diff
@@ -1153,7 +1154,7 @@ func Purge(c *db.Cluster, name string) error {
 			return fmt.Errorf("Failed to remove member %q: %w", name, err)
 		}
 
-		err = cluster.DeleteCertificates(context.Background(), tx.Tx(), name, certificate.TypeServer)
+		err = cluster.DeleteIdentitys(ctx, tx.Tx(), name, api.IdentityTypeCertificateServer)
```

Comment:
is this a DB generator bug? The incorrect pluralisation of  Identity?

Reply:
Yep, there's an issue for it here: https://github.com/canonical/lxd/issues/12815

There's also a JIRA ticket for it in my backlog.